governance/MEMBERSHIP_SURVEY.md

```diff
@@ -0,0 +1,32 @@
+
+# Membership Survey
+
+Surveys for current and past members to collect data on member wants and needs. following #375 issue.
+
+## Suggested Questions
+
+@amiller-gh
+
+What is your current happiness (perceived value?) of Ind. Membership program?
+Net promoter score – how likely would you recommend the program to a friend?
+How did you hear about the IM program?
+In your words, what value do you expect to receive from an open source individual membership?
+(For people with lapsed membership) What made you not renew?
+
+  
+
+@joesepi
+
+Why did you become an Individual Member? (If lapsed, why?)
+The goal of the Individual Membership program is to (insert clearly defined goal here); do you believe it is successful in that goal. If so/not, how/why?
```

Comment:
We may want to iterate on `(insert clearly defined goal here)`. Perhaps @dshaw, @amiller-gh, and @joesepi have more context they'd like to share.